components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIMWSDLReader.java

```diff
@@ -152,6 +152,35 @@ public OMElement readAndCleanWsdl2(API api) throws APIManagementException {
 
     }
 
+    public byte[] getWSDL() throws APIManagementException {

```

Comment:
Java doc comment need to be added for all the public methods.

Reply:
fixed in e18f43f5dee9ed43a3239af62f38d67c72c78fa2